src/main/java/org/terasology/logic/health/event/DoRestoreEvent.java

```diff
@@ -39,4 +47,12 @@ public DoRestoreEvent(int amount) {
     public int getAmount() {
         return amount;
     }
+
+    /**
+     * Method to get the instigator of restoration.
```

Comment:
Generally I'd excuse setters & getters from Javadoc unless there is any potential for confusion somehow.